webplugin/js/app/mck-app.js

```diff
@@ -377,6 +377,14 @@ function ApplozicSidebox() {
                 );
             };
 
+            //to check if the customer has been churned then show the churn banner
+            if (data.currentActivatedPlan == "churn") {
+                var churnCust = document.getElementById("km-churn-customer")
+                if (churnCust) {
+                    churnCust.classList.add("is-churn-customer")
```

Comment:
you could have use n-vis ?
This will make the code consistent

Reply:
done